frontend/src/components/account/Account.js

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from 'react';
+import EditAccountButton from '../edit-account/EditAccountButton';
+import './Account.css';
+
+
+const Account = () => {
+  const [user, setUser] = useState({});
+  const [token, setToken] = useState(window.localStorage.getItem("token"));
+  const [refresh, setRefresh] = useState(false);
+
+  useEffect(() => {
+    if(token) {
+      fetch("/accounts", {
+        headers: {
+          'Authorization': `Bearer ${token}`
+        }
+      })
+      .then(response => response.json())
+      .then(async data => {
+        window.localStorage.setItem("token", data.token)
+        setToken(window.localStorage.getItem("token"))
+        setUser(data.user)
+      })
+    }
+  }, [refresh])
+
+  const toggleRefresh = () => {
+    // sets whatever is inside refresh to the opposite, triggering a refresh
+    // through useeffect. this is activated whenever the newpostform is
+    // submitted
+    setRefresh(prevRefresh => prevRefresh + 1);
```

Comment:
if the useState is a boolean value now, this should change to:
    setRefresh(prevRefresh => !prevRefresh);

to work as intended